msg/SensorGps.msg

```diff
@@ -1,72 +1,71 @@
 # GPS position in WGS84 coordinates.
-# the field 'timestamp' is for the position & velocity (microseconds)
-uint64 timestamp		# time since system start (microseconds)
+uint64 timestamp              # Time since system start (microseconds)
 uint64 timestamp_sample
 
-uint32 device_id                # unique device ID for the sensor that does not change between power cycles
+uint32 device_id              # Unique device ID for the sensor that does not change between power cycles
 
-float64 latitude_deg		# Latitude in degrees, allows centimeter level RTK precision
-float64 longitude_deg		# Longitude in degrees, allows centimeter level RTK precision
-float64 altitude_msl_m		# Altitude above MSL, meters
-float64 altitude_ellipsoid_m	# Altitude above Ellipsoid, meters
+float64 latitude              # Latitude in degrees (allows centimeter level RTK precision)
+float64 longitude             # Longitude in degrees (allows centimeter level RTK precision)
+float64 altitude_msl          # Altitude above MSL (meters)
+float64 altitude_ellipsoid    # Altitude above Ellipsoid (meters)
```

Comment:
What ellipsoid?

Reply:
I assume EGM96? Do you think it needs to be explicitly labeled?

Reply:
Yes.